### ISSUE TITLE
mypy does not recognize a type alias to a pandas DataFrame

### ISSUE THREAD

**Issue by justinaustin** [python/mypy#9150]
Hello! It looks like mypy doesn't allow a type alias to a pandas DataFrame object in my code, but it does allow inline type hints of DataFrames.
### Are you reporting a bug, or opening a feature request?
Possibly a bug
### Please insert below the code you are checking with mypy,
  or a mock-up repro if the source is private. We would appreciate
  if you try to simplify your case to a minimal repro.
```python
import pandas as pd

Alias = pd.DataFrame

def foo(x: Alias) -> None:
    pass
```
### What is the actual behavior/output?
```text
mypy-test.py:5: error: Variable "mypy-test.Alias" is not valid as a type  [valid-type]
mypy-test.py:5: note: See https://mypy.readthedocs.io/en/latest/common_issues.html#variables-vs-type-aliases
Found 1 error in 1 file (checked 1 source file)
```
### What is the behavior/output you expect?
I expected mypy to report no errors. mypy does not report any errors for the following file:
```python
import pandas as pd

def foo(x: pd.DataFrame) -> None:
    pass
```
### What are the versions of mypy and Python you are using?
`Python 3.6.10` and `mypy 0.782`
### What are the mypy flags you are using? (For example --strict-optional)
no flags

Thank you for looking at this!

**Comment by JukkaL** [comment]
Mypy doesn't know that `pd.DataFrame` refers to a type unless you have stubs for pandas. If you don't have stubs, the alias would be equivalent to `Any` anyway, so it would be of limited use. (I'm not sure if anybody has published stubs for pandas).

If you have stubs for pandas and still see the error, they probably have an incorrect definition for `DataFrame`.
